pkg/virt-controller/services/template.go

```diff
@@ -316,14 +314,6 @@ func (t *templateService) IsPPC64() bool {
 	return t.clusterConfig.GetClusterCPUArch() == "ppc64le"
```

Comment:
Still required?

Reply:
Yes, ppc64le has some special cases in the templates. The rest of the arches was not used.